techmatters/flex-plugins
# Fetch definitions from S3 instead of bundling

## Description
Make Aselo fetch form definitions from S3 instead of bundling them.

### Related Issues
Fixes [CHI-1663](https://tech-matters.atlassian.net/browse/CHI-1663)

### Verification steps
Build `plugin-hrm-form` and check that the form definition JSONs are not bundled.
Check on the network tab that all form definitions are fetched individually.
